File: backend/springboot/src/main/java/tn/maiko26/springboot/enums/UserRole.java
package tn.maiko26.springboot.enums;

import lombok.Data;

public enum UserRole {
    ADMIN("admin"),
    USER("user");

    private final String value;

    UserRole(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: backend/dotnet/Enum/UserRole.cs
using System.ComponentModel;

namespace dotnet.Models
{
    public enum UserRole
    {
        Admin,
        User
    }

    public static class UserRoleExtensions
    {
        public static string ToStringValue(this UserRole role)
        {
            return role.ToString().ToLower();
        }
    }
}